go-controller/pkg/ovn/egressip.go

```diff
@@ -820,6 +836,26 @@ func findReroutePolicyIDs(filterOption, egressIPName string, gatewayRouterIPs []
 	return strings.Split(policyIDs, "\n"), nil
 }
 
+func findLegacyReroutePolicyIDs() ([]string, error) {
```

Comment:
comments on this function would be helpful. Mainly the difference between legacy routes and newer version routes? (multiple next hops is the only difference?)

Reply:
Yes, specifically we will be going from setting `nexthop` -> `nexthops`, so we need to clean up all the setup made w.r.t `nexthop`